src/main/features/claim/views/start.njk

```diff
@@ -10,67 +10,76 @@
   <div class="grid-row">
     <div class="column-two-thirds">
       <h1 class="heading-large">{{ t('Make a money claim') }}</h1>
+      <br>
+      <p>{{ t('You can make a money claim if a person or organisation owes you money and won’t pay you back.') }}</p>
+      <p
+        class="panel panel-border-wide">{{ t('Before claiming you should contact the person or organisation to try to find a solution. '
+        + 'Find out <a href="{{ uri }}">how you can resolve a claim by discussion or mediation..</a>',
+        { uri: '/resolve-before-claim' } ) | safe }}
+      </p>
 
-      <p>{{ t('You can make a money claim if someone owes you money and won’t agree to pay you back.') }}</p>
-
-      <p class="panel panel-border-wide">{{ t('A money claim might not be your only or best option. Find out about') }}
-        <a href="/other-options">{{ t('other ways to get your money') }}</a>.</p>
 
-      <p>{{ t('You might have to go to a hearing in front of a judge if the person says they don’t owe you. It can take
-                up to 7 months to get a hearing date.') }}</p>
+      <p>{{ t('Your claim, including your name and address, will be sent to the person you say owes you money. '
+        + 'They‘ll get a chance to respond to your claim.') }}</p>
 
-      <p>{{ t('Find out more about') }} <a
-         target="_blank" href="https://www.gov.uk/make-court-claim-for-money">{{ t('making a money claim') }}</a>,
-        {{ t('such as the') }} <a
-          target="_blank" href="https://www.gov.uk/make-court-claim-for-money/court-fees">{{ t(' fees you’ll pay') }}</a>.</p>
+      <p>{{ t('You might have to go to a hearing with a judge if the person says they don’t owe you money.') }}</p>
 
       <h2 class="heading-medium">{{ t('What you’ll need') }}</h2>
 
+      <p>{{ t('To use this service you need:') }}</p>
+
       <ul class="list list-bullet">
-        <li>{{ t('Current address and any other contact details of the person or company you’re claiming against.') }}</li>
-        <li>{{ t('A credit or debit card to pay the') }} <a
-            target="_blank" href="https://www.gov.uk/make-court-claim-for-money/court-fees">{{ t('claim fee') }}</a>.
+        <li>{{ t('current address and contact details for the person or organisation you’re claiming against (or last known address)') }}</li>
+        <li>{{ t('a credit or debit card to pay the <a href="{{ claimFeeUri }}">claim fee</a>',
+          {claimFeeUri: 'https://www.gov.uk/make-court-claim-for-money/court-fees' }) |safe }}
         </li>
-        <li>{{ t('Around 30 minutes to make your claim.') }}</li>
       </ul>
 
-      <p class="panel panel-border-wide">{{ t('You can save your claim and return to it at any time.') }}</p>
-
       <div class="clearfix">
         <a class="button button-start" href="{{ startLink }}" role="button">{{ t('Start now') }}</a>
       </div>
-
       <br>
```

Comment:
please remove this one while you're fixing the one above please :)

Reply:
done